pkg/services/featuremgmt/registry/feature_toggle.go

```diff
@@ -0,0 +1,92 @@
+package featuremgmt_registry
```

Comment:
can we use a standard package name matching the folder?

Reply:
👍  done